extensions/data-workspace/package.json

```diff
@@ -26,7 +26,7 @@
       {
         "title": "Projects",
         "properties": {
-          "dataworkspace.projects": {
+          "dataworkspace.excludedProjects": {
             "type": "array",
             "default": [],
             "description": ""
```

Comment:
This should probably have a description for users looking at the setting in the UI